src/k-provider/ott/provider-parser.js

```diff
@@ -113,6 +113,8 @@ export default class OTTProviderParser {
     const metaData = OTTProviderParser.reconstructMetadata(mediaAsset);
     metaData.description = mediaAsset.description;
     metaData.name = mediaAsset.name;
+    if (mediaAsset.data.epgId) metaData.epgId = mediaAsset.data.epgId;
+    if (mediaAsset.data.recordingId) metaData.epgId = mediaAsset.data.recordingId;
```

Comment:
```suggestion
    if (mediaAsset.data.recordingId) metaData.recordingId = mediaAsset.data.recordingId;
```
right? also need to add to the type